homeassistant/components/hvv_departures/sensor.py

```diff
@@ -0,0 +1,215 @@
+"""Sensor platform for hvv."""
+from datetime import datetime, timedelta
+import logging
+
+from aiohttp import ClientConnectorError
+from pygti.exceptions import InvalidAuth
+from pygti.gti import GTI, Auth
+
+from homeassistant.const import DEVICE_CLASS_TIMESTAMP
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
+MAX_LIST = 20
+MAX_TIME_OFFSET = 360
+ICON = "mdi:bus"
+UNIT_OF_MEASUREMENT = "min"
+
+ATTR_ATTRIBUTION = "attribution"
+ATTR_DEPARTURE = "departure"
+ATTR_LINE = "line"
+ATTR_ORIGIN = "origin"
+ATTR_DIRECTION = "direction"
+ATTR_TYPE = "type"
+ATTR_ID = "id"
+ATTR_DELAY = "delay"
+ATTR_NEXT = "next"
+
+PARALLEL_UPDATES = 0
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(
+    hass, config, async_add_entities, discovery_info=None
+):  # pylint: disable=unused-argument
```

Comment:
Why do we need to disable this?

Reply:
I think it was part of the generated code, removed it and pylint didn't complain